plugins/plugin-java/che-plugin-java-server/src/main/java/org/eclipse/che/plugin/java/languageserver/JavaLanguageServerExtensionService.java

```diff
@@ -346,6 +358,26 @@ public String getOutputDir(String projectUri) {
     }
   }
 
+  public ImplementationsDescriptorDto findImplementations(
+      FindImplementationsCommandParameters params) {
+    params.setProjectUri(LanguageServiceUtils.prefixURI(params.getProjectUri()));
+    CompletableFuture<Object> result =
+        executeCommand(FIND_IMPLEMENTORS_COMMAND, singletonList(params));
+
+    Type targetClassType = new TypeToken<ImplementationsDescriptor>() {}.getType();
+    try {
+      ImplementationsDescriptor implementationsDescriptor =
+          gson.fromJson(gson.toJson(result.get(10, TimeUnit.SECONDS)), targetClassType);
+      for (SymbolInformation symbolInformation : implementationsDescriptor.getImplementations()) {
+        Location location = symbolInformation.getLocation();
+        location.setUri(LanguageServiceUtils.removePrefixUri(location.getUri()));
```

Comment:
This will not work when the implementations are in jars. Have a look at LanguageServiceUtils#fixLocation():